app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<h2 class="center">Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" , placeholder: 'email@example.com', class:("signinField")%>
```

Comment:
- [Optional] Dear Team, You have done an incredible job with the project. The one thing I would suggest is to set up your `root route` to the login/sign-in page, or to the home page if the user is signed in. With your current configuration, I had to look for the route configuration to find the login page. Doing this phase a great advantage form a UX perspective.